Formula/vmctl.rb

```diff
@@ -0,0 +1,31 @@
+class Vmctl < Formula
+  desc "Data migration tool used to migrate data from supported DBs to VictoriaMetrics"
+  homepage "https://docs.victoriametrics.com/vmctl.html"
+  url "https://github.com/VictoriaMetrics/VictoriaMetrics.git",
```

Comment:
The answer is yes, because `victoriametrics` which you have mentioned is a time series database, when `vmctl` is a tool which provides to migrate time series metrics from other popular time series databases like Cortex, Thanos, Grafana Mimir, InfluxDB, Prometheus, OpenTSDB to VictoriaMetrics.

Reply:
I'm not sure why that means they need to be in separate formulae? Why not build `vmctl` in the `victoriametrics` formula?

Reply:
I think it is mostly user experience (assuming the releases are done at the same time for both tools), how do you handle the setup in other package managers?

Reply:
All components are shipped separately via single release - see https://github.com/VictoriaMetrics/victoriametrics-lts-rpm and https://hub.docker.com/u/victoriametrics

Reply:
> Let's just make it simpler and combine these tools in as few formulae as possible, please.

I see...
So let's say I wrote a formula which allows me to build six utilities instead of six different formulas for each utility. Do I need tests for each of the utilities? Can you give me an example of a formula where the tests cover each of them? Because the prometheus example formula is assembling the time series database, the web interface and the promtool utility, but the test is written only for promtool. What about tests for the case with six utilities?

Reply:
ping @carlocab

Reply:
any updates on this?

Reply:
Simple tests for all six would be good. If that's not possible, the more of them we're able to test, the better. I'm not sure that it's useful to look at other formulae for examples, but `llvm` ships multiple components and tests most of them.

Reply:
Ok, I'll send new PR which will contain all vmutils. So closing this PR.